From eda3b9c10e3046cf59ed74fdae9e6286bd199add Mon Sep 17 00:00:00 2001
From: Jakub Jelen <jjelen@redhat.com>
Date: Tue, 27 Oct 2026 12:35:05 +0100
Subject: [PATCH] Verify source signatures with openpgpverify

Replace the GnuPG-based %gpgverify with %openpgpverify, which uses
Sequoia sqv and takes the same options, as part of the Fedora 45 Change
https://fedoraproject.org/wiki/Changes/Sequoia_openpgpverify

Update opam-dev-pubkey.pgp to the current copy at
https://opam.ocaml.org/opam-dev-pubkey.pgp, the URL the spec names for
it (same certificate(s)).

Assisted-by: Claude Opus 5.5
---
 opam-dev-pubkey.pgp | 28 ++++++++++++++--------------
 opam.spec           |  4 ++--
 2 files changed, 16 insertions(+), 16 deletions(-)

diff --git a/opam-dev-pubkey.pgp b/opam-dev-pubkey.pgp
index 1737fc9..4c21db0 100644
--- a/opam-dev-pubkey.pgp
+++ b/opam-dev-pubkey.pgp
@@ -12,19 +12,19 @@ OMOxhuQOYV+UKN8vrs+cFicK6Ej7wgLu4eTLtp+6WGE+/ZAnfGFg3FBjuGPbd3qu
 f0CnjZbT7J83OdLWcl1kB3Tuer3Mb0HZdCBLQuY+1HulQrwK3773kvGV8HCg22+8
 M55sHeCBnt4t4NCTLAZtYbCpfFvCQsx2PmYSdCjrkhdovSOuSVgk0QwFeQARAQAB
 tB5vcGFtIGRldiB0ZWFtIDxvcGFtQG9jYW1sLm9yZz6JAlQEEwEIAD4CGwMFCwkI
-BwIGFQoJCAsCBBYCAwECHgECF4AWIQSSxSauUN85Rw6ykRvtTPHKZ8uqkgUCZsc4
-EQUJCOtzxQAKCRDtTPHKZ8uqkjOcD/4jUrmhWfvLkTz/ziY1oFxK4MRYAYBVv7Gg
-tJFNNLLPXyfNIuejtI1CufFrWHMGM6C59KguqduVYnF0NU+AK6XDMFzDTigGOrT/
-aGjw5bso+BAstUoG1huaeGcdpsrPfK18Kmb1YkTtX+052Z1XJJnIV5VyESup8nSv
-tWLSuTX9wM5fmESQmmazY+ck0q33utzH53F0SIogVzMdRoiftvzS7JrsM2jiBx3I
-BFNzz7On1fh5d/13WXENajhq1rAQzg1LXSe2lj7Q6xvjGY5A7q1H867VIa/S0+ui
-EmX7e1HQ7FF8NzhFNQqnBYMnDzO/2yPUIm6fBzdjaAu4tCsKsELsafyGggqBIiJs
-XpHEZGW1LRIKJqaeL8SqsZm+7JIMaDpticwZ46g6agPrcbo3yqXeDYTLqzzRP+gi
-qIROMQ78psGNN3CUfoYSn2TF9wD1/AcUPoXaOC+hz6yFNCcfDhCNIG/TEPpbI/OZ
-tM54P5T02q+M8G0wJ/mNFcACnVLBuQKw36JKX5Zn318PetX8Iuyq29yJM7IKQSaO
-2ki/kEvCXcW4N7/iysh/qGNMtmL5FjOCOs3xxM6JigUnURHvepDXt5D8N48O6u6c
-OtL1tuuuy/NJ5x6J8T6znmjK0wIZd5EzBkXtVJQQ9+MKuDexmXFmHJEfeTM9rctA
-61WhEfpcYLkCDQRfP7lMARAAvkMM8AYq8cAv27KGkvKanQdtdjixsqImve8c+6fg
+BwIGFQoJCAsCBBYCAwECHgECF4AWIQSSxSauUN85Rw6ykRvtTPHKZ8uqkgUCadjL
+sAUJDCSUZAAKCRDtTPHKZ8uqkgkCD/9EyVnsXeHGli7ABw/mV/1WVEatQEsklbqu
+JnWwalpQ6nAdflfzkN70FUxpunhGqnAW4Y5acM7yWv0pMItT3Ptw5Ub/fj3ufwMB
+KTcjWnghy5N+zllDHNaBQqt0q9B0nu6kyy7pUTPhUDlI0eZzzNARB3neCLZzwqdR
+FzxqvAVTPyp3eXQ0WDNDpcZBiHsHh2P2ydjaTQfngNtaArVONNsLRwJ63LWKPc4i
+KOHijHEwf9gKD8W3ZEBhpUVIw9A3uh7byFGQQZ7Tx3gkYXXBP48vBzAr/YvE582V
+FVt4yo2+3N+0WHMX4/iOayDOuo9tr8MLbwHTJDZNSFwIdSvZaSgUI/TOpPGYlAa8
+dDQxua+b4CWLa8yDTa0eyA+t1Ulk3ISUYPpAQDJaIN601ksFxihYMHBjUfbZqCKo
+3vQPybBjIGntMg4abg/88ET/In0M5uB9NBM9rs02Su9QFnUdR9z4+93/yZWL3ANe
+FloqIAxaKV4IABDB4uE+J793sCUr9OlkpEJXcUCm2XlbP/I6/lDfXe28N1ZYcr5o
+w2VstqW+zPK/6NeG6SGAEU8+ziCP5hGLOcCgxZZnXwrzzGq1+Yc9iaJEYYT9OEvO
+p/0emBCwdWwQ3BLrn+w55I8WdZXrbowar6XqlfiOWbILRLDIq66N5r/PboQ20mad
+L7EbgGVIY7kCDQRfP7lMARAAvkMM8AYq8cAv27KGkvKanQdtdjixsqImve8c+6fg
 ReQwyKxQkTIIjGee5YVzpYNEzHKp8oyzwib4KB0XO51ZdHw+aaizAT+pHfWCFR3K
 DBDBfIsLtoeiO/t/Y/XkoR+RfDYdzi3x/Ys2YcTtfY3QlS6mKN1gOVIA9IlRHuw9
 wyO4P2JWJzQeSqU8SmKwCDfztBRwuZR2pbAnYyGVEzUP+XxrHbl0/2TdDogTPRRN
@@ -48,5 +48,5 @@ HidQM53CXRgCEfNd5Gw5dFeiQmFrL1loG3Wd7aV2gw7P+yErjdgJKJCXCqXFvOCe
 f/zxoFHWDS4a/camY9uvgSGGNwZryDBzmCYHWBr+LDIh0nzsZTLoPwCyGChsTDc3
 goOJb8wAlp2omYfNF0JCHASCxC+uTpZixUxGZWz8vZVcgBV6mCAuOsdcLxD8XXja
 yj8YRAw4
-=0d3o
+=uaWX
 -----END PGP PUBLIC KEY BLOCK-----
diff --git a/opam.spec b/opam.spec
index ac4f08a..c6ddf18 100644
--- a/opam.spec
+++ b/opam.spec
@@ -38,7 +38,7 @@ Patch:          %{name}-disable-tests.patch
 BuildRequires:  diffutils
 BuildRequires:  gcc-c++
 BuildRequires:  git-core
-BuildRequires:  gpgverify
+BuildRequires:  openpgpverify
 BuildRequires:  make
 BuildRequires:  ocaml >= 4.11.0
 BuildRequires:  ocaml-base64-devel >= 3.1.0
@@ -107,7 +107,7 @@ Standalone script for working with opam .install files, see the opam package
 for more information.
 
 %prep
-%{gpgverify} --data=%{SOURCE0} --signature=%{SOURCE1} --keyring=%{SOURCE2}
+%{openpgpverify} --data=%{SOURCE0} --signature=%{SOURCE1} --keyring=%{SOURCE2}
 %autosetup -n %{name}-full-%{version} -p1
 sed -e 's,%%SOURCE3%%,%{SOURCE3},' \
     -e 's,%%SOURCE4%%,%{SOURCE4},' \
-- 
2.56.0

